product-features/liquidity-providers/README.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Participate in ALEX DEX liquidity pools and earn a share of the trading fees!
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: false
+---
+
+# 🐋 Liquidity Providers
+
+By adding liquidity to a pool, you will earn fees from all trades between a token pair proportional to your share of the pool. Fees are added to the pool, automatically accrued in real-time and can be claimed by withdrawing your liquidity.
```

Comment:
Mhm... I am unconvinced about that, since what are "claimed" in this sentence are the fees, and fees are new money for the user.  Maybe I can add the "back" work after "withdrawing", since the liquidity was originally form the user.

> By adding liquidity to a pool, you will earn fees from all trades between a token pair proportional to your share of the pool. Fees are added to the pool, automatically accrued in real-time and can be claimed by withdrawing back your liquidity.

I will commit with this change, let me know if I should address any other change.